aries-site/src/examples/components/dateinput/DateInputValidationExample.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Box, Form, FormField, DateInput, Text } from 'grommet';
+import PropTypes from 'prop-types';
+import { CircleAlert } from 'grommet-icons';
+
+const Error = ({ children, ...rest }) => {
+  return (
+    <Box direction="row" {...rest}>
+      <Box flex={false} margin={{ top: 'xxsmall', right: 'xxsmall' }}>
+        <CircleAlert size="small" />
+      </Box>
+      <Text size="xsmall">{children}</Text>
+    </Box>
+  );
+};
+
+Error.propTypes = {
+  children: PropTypes.string,
+};
+
+export const DateInputValidationExample = () => {
+  const [value, setValue] = React.useState('');
+  const [message, setMessage] = React.useState(
+    <Error background="background-front">Select date to resolve error</Error>,
+  );
+  const onChange = event => {
+    const nextValue = event.value;
+    console.log('onChange', nextValue);
+    setValue(nextValue);
+    setMessage('');
+  };
+  return (
+    <Box align="center" pad="large">
+      <Form>
+        <FormField
+          name="required-field"
```

Comment:
Should the name be matching the name below?

Reply:
good catch thank you!